src/main/java/dev/corestone/lotrings/abilities/LightningAbility.java

```diff
@@ -0,0 +1,65 @@
+package dev.corestone.lotrings.abilities;
+
+import dev.corestone.lotrings.LOTRings;
+import dev.corestone.lotrings.Ring;
+import dev.corestone.lotrings.RingState;
+import dev.corestone.lotrings.abilities.abilityutil.CooldownManager;
+import org.bukkit.Sound;
+import org.bukkit.block.Block;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.HandlerList;
+import org.bukkit.event.player.PlayerInteractEvent;
+import org.bukkit.util.RayTraceResult;
+
+public class LightningAbility extends AbilitySuper {
+
+    private double range;
+    private CooldownManager cooldownManager;
+    private Sound sound;
+
+    public LightningAbility(LOTRings plugin, Ring ring, String abilityName) {
+        super(plugin, ring, abilityName);
+        try {
+            this.range = plugin.getAbilityDataManager().getAbilityFloatData(abilityName, "range").doubleValue();
+            this.cooldownManager = new CooldownManager(plugin, this, plugin.getAbilityDataManager().getAbilityFloatData(abilityName, "cooldown-seconds").doubleValue());
+            this.sound = Sound.valueOf(plugin.getAbilityDataManager().getAbilityStringData(abilityName, "sound").toUpperCase());
+        } catch (Exception e) {
+            sendLoadError();
+        }
+    }
+
+    @Override
+    public void switchState(RingState ringState) {
+        if (ringState == RingState.LOST) {
```

Comment:
this function is not needed if you're only unregistering. You can leave it or deleted idc.